types/index.d.ts

```diff
@@ -0,0 +1,24 @@
+export = parserWrapper;
```

Comment:
Well,

This is not correct declaration file:

  1. `defaultDirectives` available in the `options` interface – https://github.com/posthtml/posthtml-parser/blob/master/index.js#L19.
  2. `parserWrapper` signature has `options` parameter – https://github.com/posthtml/posthtml-parser/blob/master/index.js#L28.
  3. Instead use `PostHTML` as prefix in the interface name you can use real `namespace`.
  4. The properties in the `defaultOptions` is optional properties.

So, correct `.d.ts` might look like this:

```ts
declare function parserWrapper(html: string, options?: PostHTMLParser.IOptions): PostHTMLParser.INode[];

declare namespace PostHTMLParser {
    export interface INode {
        tag: string;
        attrs: Record<string, string>;
        content: Array<string | INode>;
    }

    export interface IOptions {
        lowerCaseTags?: boolean;
        lowerCaseAttributeNames?: boolean;
        directives?: IDirective[];
    }

    export interface IDirective {
        name: string;
        start: string;
        end: string;
    }
}

export = PostHTMLParser;
```